src/views/apply/index.html

```diff
@@ -3,6 +3,19 @@
 {% set homeLink = true %}
 {% set footer = true %}
 {% block pageHeading %}Apply to Hack Cambridge{% endblock %}
+{% block introBanner %}
+    <section class="red">
+        <div class="container a4">
+            <div class="one half unit row">
+                <div class="unit column grows">
+                    <div>
+                        <a href="../" rel="prev">Find out more about Hack Cambridge</a>
```

Comment:
Use absolute link i.e. `/` — then if this is included in a different path it won't break